src/middlewared/middlewared/plugins/audit/audit.py

```diff
@@ -129,7 +129,7 @@ async def compress(self, data):
 
     @accepts(Dict(
         'audit_query',
-        List('services', items=[Str('db_name', enum=ALL_AUDITED)], default=ALL_AUDITED),
+        List('services', items=[Str('db_name', enum=ALL_AUDITED)], default=['MIDDLEWARE', 'SUDO']),
```

Comment:
Nothing _wrong_ with this, but maybe could make a little more future-proof (so that it won't break when additional services are added).  For example...
```
ALL_AUDITED = [svc[0] for svc in AUDITED_SERVICES]
BULK_AUDIT = ['SMB']
NON_BULK_AUDIT = [svc for svc in ALL_AUDITED if svc not in BULK_AUDIT]
...
 List('services', items=[Str('db_name', enum=ALL_AUDITED)], default=NON_BULK_AUDIT),
```

Meh.